src/main/webapp/pages/analyses/analyses-table.html

```diff
@@ -126,8 +126,8 @@ <h3 class="popover-title"></h3>
           base: /*[[@{/}]]*/ '/',
           analyses: /*[[@{${ajaxURL}}]]*/ "",
           analysis: /*[[@{/analysis/}]]*/ "",
-          download: /*[[@{/analysis/ajax/download/}]]*/ "",
-          deleteUrl: /*[[@{/analysis/ajax/delete/}]]*/ ""
```

Comment:
I like to build the urls in the api file itself using the `window.TL.BASE_URL` . why do you do this on the page.